From 524295973653e269ee74ca609de14772cce7553b Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 25 Apr 2017 13:47:50 -0700
Subject: [PATCH] CMON-64 hermes should upload "cmon" service logs Reviewed by:
 Richard Kiene <richard.kiene@joyent.com> Approved by: Richard Kiene
 <richard.kiene@joyent.com>

---
 etc/logsets.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/logsets.json b/etc/logsets.json
index f019b96..929fd32 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -213,6 +213,7 @@
       "binder",
       "ca",
       "cloudapi",
+      "cmon",
       "cnapi",
       "cns",
       "dhcpd",
-- 
2.21.0

